source/excerptexport/templates/excerptexport/templates/index.html

```diff
@@ -1,6 +1,12 @@
 {% extends "excerptexport/layouts/base.html" %}
 
 {% block main %}
+    {% if user.is_authenticated %}
+    <a href="{% url 'excerptexport:logout' %}?next={% url 'excerptexport:index' %}">Logout <strong>{{ user.username }}</strong></a>
```

Comment:
Even if `flake8` only checks Python files, please limit line length to 119 characters in other files, too.